docs/releases/earlier_releases.md

```diff
@@ -320,7 +320,7 @@ Besides lots of smaller improvements and fixes found bellow, and mentioned above
 
 Symfony 2.7 LTS is now fully supported, and no deprecation errors should be thrown anymore. This should also ensure compatibility with the 2.8 and 3.0 releases planned for November this year.
 
-Dynamic settings have been refactored to use the [Expression Language](http://symfony.com/fr/doc/current/components/expression_language/index.html) instead of fake services.
+Dynamic settings have been refactored to use the [Expression Language](http://symfony.com/doc/current/components/expression_language/index.html) instead of fake services.
```

Comment:
```suggestion
Dynamic settings have been refactored to use the [Expression Language](http://symfony.com/doc/4.3/components/expression_language/index.html) instead of fake services.
```